cql3/statements/alter_table_statement.cc

```diff
@@ -398,19 +399,45 @@ alter_table_statement::prepare_schema_mutations(query_processor& qp, api::timest
 
 std::unique_ptr<cql3::statements::prepared_statement>
 cql3::statements::alter_table_statement::prepare(data_dictionary::database db, cql_stats& stats) {
+    // Cannot happen; alter_table_statement is never instantiated as a raw statement
+    // (instead we instantiate alter_table_statement::raw_statement)
+    utils::on_internal_error("alter_table_statement cannot be prepared. Use alter_table_statement::raw_statement instead");
```

Comment:
Somewhat sad that we have to insert new internal errors. But I understand where it comes from - parts of the hierarchy are split into raw and prepared, others aren't.

Ideally we should split everything, but that's very tedious.